github_search/lib/presentation/github_repository_detail_screen/widget/repository_detail_body_item_widget.dart

```diff
@@ -10,17 +10,19 @@ class RepositoryDetailBodyItemWidget extends StatelessWidget {
     super.key,
     required this.repository,
     required this.icon,
+    required this.getRepository
   });
 
   final GitHubRepository repository;
+  final GitHubGetRepository? getRepository;
   final RepositoryItemIcon icon;
 
   Widget countText(RepositoryItemIcon icon) {
     switch (icon) {
       case RepositoryItemIcon.stars:
         return Text(repository.starCountWithComma);
       case RepositoryItemIcon.watchers:
-        return Text(repository.watchCountWithComma);
+        return Text(getRepository?.subscribersCountWithComma ?? "0");
```

Comment:
Not "0" but "-" use. Because it's possible to get "0" as success.

Reply:
fixed: https://github.com/shusuke0812/flutter-engineer-codecheck/pull/5/commits/9fca9a3b7b5441a93870d4134b4337b8fa2ffe48